src/main/java/andersen/AndersenTasks/BusTicket.java

```diff
@@ -0,0 +1,15 @@
+package andersen.AndersenTasks;
+
+import lombok.Data;
+
+@Data
+public class BusTicket {
+
+    String ticketClass;
```

Comment:
The rule of thumb is always use private access modifier for variables, or. in this case you could think about java records